includes/managers/controls.php

```diff
@@ -941,6 +941,12 @@ public function add_page_transitions_controls( Controls_Stack $controls_stack, $
 			__( 'Page Transitions let you style entrance and exit animations between pages as well as display loader until your page assets load.', 'elementor' ),
 		];
 
+		// If the user has a Pro but the experiment is off.
+		// (Don't need to check if the experiment is off because if it's on, the Pro will override this section anyway).
+		if ( Utils::has_pro() ) {
+			$messages[] = __( 'This feature is currently an experiment, you can turn it on in Elementor --> Settings --> Experiments.', 'elementor' );
```

Comment:
Add a link with target=_blank. but exclude the HTML tag from the translation.
```
sprintf( __( 'This feature is currently an experiment, you can turn it on in %s' ), sprintf( '<a...</a>', __( 'Elementor' ), __( 'Settings' )...  )
```

Reply:
I was actually about to do that, IDK why I didn't 😅